test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
What is this for?

Reply:
tests auto-generate because of a gem. don't worry about it rn!